build.sbt

```diff
@@ -65,6 +70,15 @@ val coreDependencies = Seq(
   "javax.validation" % "validation-api" % "1.1.0.Final" force()
 )
 
+val loggingDependencies = Seq(
+  "ch.qos.logback" % "logback-core" % "1.1.2",
+  "ch.qos.logback" % "logback-classic" % "1.1.2",
```

Comment:
I think it makes sense to only have it in the server. In core tranquility it often conflicts with other loggers people want to use (as tranquility is always embedded in some larger app).

But, server is its own standalone app, so it needs a logger.